### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by phylischumba**
### Milestone One

- Delete any unnecessary file or asset
- Add prop-types and big.js using npm
- Run the application with npm start and make sure that everything works correctly.
- Initialize a git repo
- Deploy to Heroku

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
-# Calculator-React
\ No newline at end of file
+# Calculator-React
```
Comment (bellom) [inline README.md]: **Let's make your repo looks professional by applying all the 3 basic rules, kindly update your `README` file and remove unnecessary content**.
Checkout out the template for help, free free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Phylis05  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by phylischumba** [comment]
Hey @bellom, thanks for the review. I have implemented the changes requested.

**Review by Oghenebrume50 — APPROVED** [review]
### APPROVED 

Good job making changes @Phylis05  the project could actually use some cleanup in the readme and be more professional but this is good for now :slightly_smiling_face: Please merge and move on **This is approved** :+1: 

**RAPHAEL NORIODE** :mask: STAY_SAFE

**Comment by phylischumba** [comment]
Thank you @Oghenebrume50. I will look into it as I proceed

**Pull request merged**